src/utils.rs

```diff
@@ -323,6 +325,24 @@ pub fn selection_ranges(root: &SyntaxNode, content: &str, pos: Position) -> Opti
     root.map(|b| *b)
 }
 
+pub fn atom_edit_to_tuple(atom_edit: AtomEdit) -> (ops::Range<usize>, String) {
+    let r: std::ops::Range<usize> = atom_edit.delete.start().into()..atom_edit.delete.end().into();
```

Comment:
Are the two dots really needed here or is that a Rust feature I'm not aware of? :)

Reply:
The `..` operator creates a `Range` from two numbers. Since `AtomEdit`'s `delete` property is of type `TextRange`, which belongs to the `text-size` crate, I'm converting that range type to the builtin `std::ops::Range` type to avoid adding dependencies to `textedit-merge`.